src/items/menu.js

```diff
@@ -109,6 +109,31 @@ const makeTree = (core, __, metadata) => {
  */
 export default class MenuPanelItem extends PanelItem {
 
+  init(state, actions) {
+    const onkeydown = ev => {
+      const checkKeys = (this.options.boundKey || 'Alt+a').toLowerCase().split('+');
+      const modifierNames =  ['ctrl', 'shift', 'alt', 'meta'];
+      const keyName = String(ev.key).toLowerCase();
+      const validKeypress = checkKeys.every(k => modifierNames.indexOf(k) !== -1
+        ? ev[k + 'Key']
+        : keyName === k);
+
+      if (!validKeypress) {
+        return;
+      }
+
+      this.panel.$element
+        .querySelector('.osjs-panel-item[data-name=menu] > d')
```

Comment:
You have a typo there. It says `d` not `div` at the end.

But I have figured out a way to do this without resorting to `querySelector`, which is kind of an anti-pattern anyway.

If you create a new method:

```
attachKeybindings(el) {
  const onkeydown = ev => {
    // .. code ..

   if (validKeypress) {
      el.click();
   }
  };

  // .. rest of the code you had previously .. //
}
```

And then in the render method change to:

```
    return super.render('menu', [
      h('div', {
        onclick,
        oncreate: el => this.attachKeybindings(el), // <-- NEW LINE
        style: {
          backgroundImage: `url(${menuIcon})`
        }
      }, h('span', {}, _('LBL_MENU')))
    ]);
```

:tada: No manual DOM lookups and it will work for multiple items!